### DISCUSSION TITLE
Remove certificate template from course batch not working

### DISCUSSION THREAD

**Discussion by pranavirabatti** [Sunbird-Lern/Community#28] [Issues] [answered]
**Sunbird portal version 4.4.1** 

http://localhost:3000/api/course/batch/cert/v1/template/remove

Response:

```
{
    "id": "api.course.batch.cert.template.remove",
    "ver": "v1",
    "ts": "2022-07-22 17:17:36:032+0000",
    "params": {
        "resmsgid": null,
        "msgid": "c13e2253-94c4-dbe1-555d-d82d62fdecf5",
        "err": null,
        "status": "SERVER_ERROR",
        "errmsg": null
    },
    "responseCode": "SERVER_ERROR",
    "result": {}
}
```

Logs:
https://pastebin.com/JXe43DqW

@neha0305verma @rahul-tarento @niharikasingh84 @pc0202 can you please help me here?

**Comment by rhwarrier** [comment, marked as answer]
@reshmi-nair  could you check on this please ?

**Reply by pranavirabatti** [reply]
Any update on this? Please help us here

**Reply by reshmi-nair** [reply]
@pranavirabatti There is a trigger on course_batch table in sunbird_courses keyspace. 
course_batch_trigger ON sunbird_courses.course_batch USING 'org.sunbird.cassandra.triggers.TransactionEventTrigger'. 

Please delete this trigger to solve the issue, also check if all the nodes in Cassandra cluster are working or not.

**Reply by pranavirabatti** [reply]
Ok @reshmi-nair I will check this and update

**Reply by pranavirabatti** [reply]
Thank you so much @reshmi-nair
